examples/acados_python/tests/soft_constraint_test.py

```diff
@@ -61,23 +61,14 @@ def run_closed_loop_experiment(FORMULATION):
     N = 20
 
     # set dimensions
-    ocp.dims.nx  = nx 
-    ocp.dims.ny  = ny 
-    ocp.dims.ny_e = ny_e 
-    ocp.dims.nu  = model.u.size()[0]
     ocp.dims.ns = nu 
```

Comment:
maybe add a comment on why these dims are specified and others not?

Reply:
actually also `ns*` are detected. addressed..